config/example/config.toml

```diff
@@ -44,7 +44,7 @@
 # Set this to true if you are using database-based admins, or false if you want to define them in this file
 # If the database fails, this file will be used as fallback
 use_database_admins = false
-# Auto authenticate localhost users as admin? Useful for test servers, disable in producation
+# Auto authenticate localhost users as admin? Useful for test servers, disable in producation67
```

Comment:
Shouldn't this be `production`?

Reply:
Yes, sorry mistype there! How do I fix this?I am a bit unsure of how to fix this in this pull request

Reply:
Just correct the typo in whatever program you used to make the code changes, save, and then push it to your remote. The PR will update with the new commit.